src/event.cpp

```diff
@@ -60,7 +63,6 @@ std::string enum_to_string<event_type>( event_type data )
         case event_type::npc_becomes_hostile: return "npc_becomes_hostile";
         case event_type::opens_portal: return "opens_portal";
         case event_type::opens_temple: return "opens_temple";
-        case event_type::player_levels_spell: return "player_levels_spell";
```

Comment:
This name was wrong, but I hadn't fixed it yet because it will require migration code.  If you want to implement such migration code, that would be great :).